KNOWNPROBLEMS.md

```diff
@@ -1,18 +1,27 @@
-# exampleClient Known Problems
+# Known Problems
 
-## BaseV3Channel.java
+## helloRPC, rdbService, and serviceAPI
+
+These are left almost unchanged since release 4.5.
+
+**Greg:** Please look at these examples and decide what should be done.
```

Comment:
Please open an issue for this.
I don't think a remark in a known problems file is enough to trigger an action of someone else.